.github/workflows/pip-build-osx.yml

```diff
@@ -3,8 +3,8 @@ name: pip build osx
 on: [push, pull_request]
 
 env:
-  MACOSX_DEPLOYMENT_TARGET: 10.14
-  _PYTHON_HOST_PLATFORM: macosx-10.14-universal2
+  MACOSX_DEPLOYMENT_TARGET: 10.15
+  _PYTHON_HOST_PLATFORM: macosx-10.15-universal2
```

Comment:
Don't we get warnings if this doesn't match -mmacosx-version-min in src/python/CMakeLists.txt? (I don't remember exactly)